spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/util/URLUtil.java

```diff
@@ -31,31 +29,76 @@
  **/
 public class URLUtil {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(URLUtil.class);
-
   private static final String SCHEMA_SEPRATOR = "://";
 
+  private static final String IPPORT_SEPRATOR = ":";
+
   private static final String SYSTEM_KEY_BOTH = "PAAS_CSE_ENDPOINT";
 
   private static final String SYSTEM_KEY_SERVICE_CENTER = "PAAS_CSE_SC_ENDPOINT";
 
   private static final String SYSTEM_KEY_CONFIG_CENTER = "PAAS_CSE_CC_ENDPOINT";
 
+  /**
+   * @param url
+   * @return
+   */
+  public static String[] splitIpPort(String url) {
```

Comment:
```http://localhost/``` should be a valid url

Reply:
the port is not given, how to support

Reply:
http default to 80, https default to 443

Reply:
method splitIpPort() only deals with urls  returned by the server center. 
server center will not return url like 'http://localhost/'.